src/OpenTelemetry.Exporter.Prometheus.HttpListener/OpenTelemetry.Exporter.Prometheus.HttpListener.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <Description>Prometheus exporter HttpListener for OpenTelemetry .NET</Description>
```

Comment:
nit: Maybe...

`Stand-alone HttpListener for hosting OpenTelemetry .NET Prometheus exporter`

🤷‍♀️